tools/toolchains/arm.py

```diff
@@ -429,6 +461,8 @@ class ARMC6(ARM_STD):
         "Cortex-A9"
     ]
     ARMCC_RANGE = (LooseVersion("6.10"), LooseVersion("7.0"))
+    MBED_STUDIO_LICENSE_FILE_NAME = "ac6-license.dat"
```

Comment:
Same here. Hardcoding the name will also limit any options of changing the file name of its location. I think it is worth revisiting if there is any other option of doing it.

Reply:
At some point the Mbed OS tools need to know what license file to pass to the compiler. So we need either a file to always pass to the compiler, or we would need some kind of file (in a location that doesn't move) that we could parse to then get the location of the license file/directory. Since 0.4.0 has already shipped without such a file (to my knowledge), I think we probably need to go with what's currently implemented. Though again, I'm open to suggestions!

Reply:
Please consider for the future that if this version of ARMC6 and license are made available for use outside Mbed Studio our team will probably be doing so.

We ensure all our devs are using the same tool-chain by containerizing it.